javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,20 @@
+var fullName = 'Heesoo Shane Kim';
+console.log('value of fullName', fullName);
+console.log('typeof fullName', typeof 'Heesoo Shane Kim');
+
+var isCool = true;
+Boolean(isCool);
+console.log('value of isCool', isCool);
+console.log('typeof isCool', typeof true);
+
+var TotalPets = 0;
```

Comment:
Variables should not start with a capital letter, that means something very specific you will learn about in the future. This should be `totalPets`